env.default

```diff
@@ -33,9 +34,7 @@ DISABLE_MUTUALIZED_PRINT=#
 DISABLE_LOCAL_PRINT=
 # Set DISABLE_HAPROXY_LOGGING to # to disable logging
 DISABLE_HAPROXY_LOGGING=
-HAPROXY_LOGGING_OPTIONS="log global
-    option httplog
-    option dontlognull"
+HAPROXY_LOGGING_OPTIONS="log global\noption httplog\noption dontlognull"
```

Comment:
Keep multilines, thaks :-)

Reply:
sorry, I commited the file by mistake, I believe.
Thanks